src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,103 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int treshhold;
```

Comment:
The spelling of the variable name 'treshhold' is incorrect. It should be 'threshold'. It's a common mistake, but it's important to use the correct spelling for variable names as it improves code readability.